src/components/Accordion.js

```diff
@@ -0,0 +1,93 @@
+const menuOpen = "collapse show";
+const buttonOpenAll = "Open All";
+const buttonCloseAll = "Close All";
+
+if (!String.prototype.includes) {
+  String.prototype.includes = function(search, start) {
+    "use strict";
+    if (typeof start !== "number") {
+      start = 0;
+    }
+
+    if (start + search.length > this.length) {
+      return false;
+    } else {
+      return this.indexOf(search, start) !== -1;
+    }
+  };
+}
+
+document.addEventListener(
+  "click",
+  clickEvent => {
+    const { target } = clickEvent;
+
+    if (target && target.className === "dg_allitems") {
+      AllMenuItemsAction(target);
+    } else if (target) {
+      MenuAction(target);
+    } else {
+      return;
+    }
+  },
+  false
+);
+
+function MenuAction(element) {
+  var mainDiv = element.parentElement.parentElement;
+  var menuItems = mainDiv.getElementsByClassName("multi-collapse");
+  var childDiv = element.nextElementSibling;
+  var menuID = childDiv.id;
+
+  for (i = 0; i < menuItems.length; i++) {
+    if (menuID !== menuItems[i].id) {
+      menuItems[i].className = mainDiv.className.includes(
+        "dg_allowmutlipleopen"
+      )
+        ? menuItems[i].className
+        : menuState("close");
+    } else {
+      if (menuItems[i].className.includes(menuOpen)) {
+        menuItems[i].className = menuState("close");
+        menuItems[i].setAttribute("aria-expanded", false);
+        element.parentElement.className = "card fa collapsed dg_menuitem";
+      } else {
+        menuItems[i].className = menuState("open");
+        menuItems[i].setAttribute("aria-expanded", true);
+        element.parentElement.className = "card fa dg_menuitem";
```

Comment:
let's get anything with cards out of here.

Reply:
also, what is `fa` doing here?

Reply:
fa removed

Reply:
fa was removed from that element but I had to add it back to the button css class or else the arrow doesn't show. Not sure why that's the case since I'm importing all font awesome css but for now I'll leave it in the html. During prototype we can re-address and remove as needed.